doc/nrf/ug_thread_ot_integration.rst

```diff
@@ -0,0 +1,130 @@
+.. _openthread_integration:
+
+OpenThread Integration
+######################
+
+.. contents::
+    :local:
+    :depth: 2
+
+The goal of this page is to explain how the OpenThread stack is integrated with remaining parts of the system in terms of the file structure,
+used threads and synchronization primitives and the network traffic flow.
+
+The OpenThread stack is integrated with the Zephyr RTOS as the L2 layer with the IEEE 802.15.4 Radio Driver underneath and Zephyr IPv6 stack above it.
+The main advantage of this approach is that it can utilize the Zephyr's L3 layer which allows for easy easy application porting and utilization of UDP, TCP and ICMP build on top of Zephyr's IPv6 stack.
```

Comment:
```suggestion
The advantage of using the Zephyr IPv6 stack is that it can utilize the Zephyr's L3 layer which allows for application portability and utilization of UDP, TCP and ICMP built on top of Zephyr's IPv6 stack.
```